### ISSUE TITLE
Keyboard shortcuts for changing dev tool gizmo modes in desktop mode

### ISSUE THREAD

**Issue by JackTheFoxOtter** [Yellow-Dog-Man/Resonite-Issues#3219]
### Is your feature request related to a problem? Please describe.

Currently, when working with the dev tool, to change the gizmo mode (for example between rotation / translation / scale), you need to use the context menu, which creates friction and slows down desktop world building workflows.

### Describe the solution you'd like

Keyboard shortcuts to allow quickly cycling / changing between the common gizmo modes when using a dev tool.

### Describe alternatives you've considered

Not doing this (I can't really think of an alternative, I guess on screen buttons but that would probably be much more work).

### Additional Context

I'm requesting this for a friend of mine who's a level designer and used to working in other game engines. This is one of the largest friction points with Resonite's building tools he and other game dev contacts of his have when trying to create content for it.

### Requesters

spacelute on Discord

**Comment by JackTheFoxOtter** [comment]
I didn't find an existing option yet. I will have to look into that tomorrow. That would definitely help in the meantime! But I still believe there should be native keybinds.

**Comment by epicEaston197** [comment]
> I didn't find an existing issue yet.

While my issue that I created wasn't exactly what you were looking for the intent behind creating this issue is the exact same I wanted to be able to switch gizmo options with buttons https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/1485

**Comment by Dusty-Sprinkles** [comment]
The difference here could probably be that this might be a content issue instead of existing functionality being requested, just having them added to the dev tool as offical hotkeys probably

**Comment by JackTheFoxOtter** [comment]
Agreed, that could be a potential solution to this. It's up for the team to decide which route works best here, I just reported the problem.

**Comment by epicEaston197** [comment]
I think this is a content issue

**Comment by Shywizz** [comment]
I did this on my dev tool, Q and E to swap gizmo, among some other niceties if anyone reading this is interested

**Comment by Redd56** [comment]
another thing is extra gizmo modes like primitive colliders and procedural mesh's they have gizmos as well and one to go up a slot. and these can access via the gizmo submenu with the devtool, a button to open a menu that shows all these other gizmo butons would be nice
